### PR TITLE
Draw smooth line with specific thickness in Tracer

### LINKED ISSUE #264: Improvement suggestions for the Tracers module
**Is your feature request related to a problem? Please describe.**
No, just cosmetic improvements.

**Describe the solution you'd like**
I would like to suggest the following cosmetic improvements to the Tracers module:
- Smoothing in the lines used, because currently they look like a staircase;
- Thickness option for the lines.

**Describe alternatives you've considered**
Continue using the tracers as is.

**Additional context**
No.

### PR DESCRIPTION
**Pull request by Luna5ama**
closes #264

### CODE REVIEW

File: src/main/kotlin/me/zeroeightsix/kami/feature/module/Tracers.kt [hunk 1]
```diff
@@ -38,8 +34,15 @@ object Tracers : Module() {
     @Setting
     private var range = 200.0
 
+    @Setting
+    private var thickness: @Setting.Constrain.Range(
+    min = 0.1,
+    max = 8.0,
+    step = 0.1
+    ) Float = 1.5f
```
Comment (zeroeightysix) [inline src/main/kotlin/me/zeroeightsix/kami/feature/module/Tracers.kt]: Can you fix the indent here?

File: src/main/kotlin/me/zeroeightsix/kami/feature/module/Tracers.kt [hunk 2]
```diff
@@ -38,8 +34,15 @@ object Tracers : Module() {
     @Setting
     private var range = 200.0
 
+    @Setting
+    private var thickness: @Setting.Constrain.Range(
+    min = 0.1,
+    max = 8.0,
+    step = 0.1
```
Comment (5HT2) [inline src/main/kotlin/me/zeroeightsix/kami/feature/module/Tracers.kt]:
```suggestion
        min = 0.1,
        max = 8.0,
        step = 0.1
```

**Review by 5HT2 — CHANGES_REQUESTED** [review]

**Comment by 5HT2** [comment]
```diff
kami-fabric % ktlint -F src/main/kotlin/me/zeroeightsix/kami/feature/module/Tracers.kt 
/home/mika/projects/kami-fabric/src/main/kotlin/me/zeroeightsix/kami/feature/module/Tracers.kt:17:1: Wildcard import (cannot be auto-corrected)
kami-fabric % git diff
diff --git a/src/main/kotlin/me/zeroeightsix/kami/feature/module/Tracers.kt b/src/main/kotlin/me/zeroeightsix/kami/feature/module/Tracers.kt
index fd30ed1d..5a1295b6 100644
--- a/src/main/kotlin/me/zeroeightsix/kami/feature/module/Tracers.kt
+++ b/src/main/kotlin/me/zeroeightsix/kami/feature/module/Tracers.kt
@@ -36,9 +36,9 @@ object Tracers : Module() {
 
     @Setting
     private var thickness: @Setting.Constrain.Range(
-    min = 0.1,
-    max = 8.0,
-    step = 0.1
+        min = 0.1,
+        max = 8.0,
+        step = 0.1
     ) Float = 1.5f
 
     @EventHandler
@@ -91,4 +91,4 @@ object Tracers : Module() {
         glDisable(GL_LINE_SMOOTH)
         glHint(GL_LINE_SMOOTH_HINT, GL_DONT_CARE)
     })
-}
+}
\ No newline at end of file
```
That should be the only style issue according to ktlint, maybe a ktlint gradle plugin would help with reviewing PRs

**Review by zeroeightysix — APPROVED** [review]

**Comment by zeroeightysix** [comment]
Thanks!

**Pull request merged**